src/javascripts/ng-admin/Crud/button/maListButton.js

```diff
@@ -21,7 +21,13 @@ define(function () {
                 scope.label = scope.label || 'List';
 
                 scope.gotoList = function () {
-                    $state.go($state.get('list'), { 'entity': scope.entity().name() });
+                    $state.go($state.get('list'), {
+                        entity: scope.entity().name(),
```

Comment:
Doesn't make sense for the list button IMO. If I add a list button to an existing list view, it should lead to another list view and therefore have empty params.